src/Header.tsx

```diff
@@ -22,33 +22,58 @@ function EblLogo(): JSX.Element {
   )
 }
 
-function NavItem(props: { href: string; title: string }): JSX.Element {
+function NavItem(props: {
+  href: string
+  title: string
+  eventKey: string
+}): JSX.Element {
   return (
     <Nav.Item>
       <LinkContainer to={props.href}>
-        <Nav.Link>{props.title}</Nav.Link>
+        <Nav.Link eventKey={props.eventKey}>{props.title}</Nav.Link>
```

Comment:
Is the `eventKey` required? [`activeKey` should also match the `href`.](https://react-bootstrap.github.io/components/navs/#nav-props)

Reply:
even better